### PR TITLE
dev: 234 separate test improvementsextension from 181

### PR DESCRIPTION
**Pull request by ArdtK**
## Issue addressed
Solves #234 

## Code of conduct
- [x] I HAVE NOT added sensitive or compromised (test) data to the repository.
- [x] I HAVE NOT added vulnerabilities to the repository.
- [x] I HAVE discussed my solution with (other) members of the KOSWAT team.

## What has been done?
- Add `dike_section` to shp-export and to names of layer images (will change **MANY** test result files)
- Add additional dikes and scenarios

### Checklist
- [x] Tests are either added or updated.
- [x] Branch is up to date with `master`.
- [x] Updated documentation if needed.

## Additional Notes (optional)
- ~~Some changes to shorten the paths of test output files (gives issue with pytest)~~
- Move logic to consolidate the export path/file name to `ReinforcementProfile`
- Extend test result sync script to also sync data for new folders

### CODE REVIEW

File: tests/acceptance_scenarios/layers_cases.py [hunk 1]
```diff
@@ -47,8 +47,6 @@ class LayersCases(CasesProtocol):
 
     # Only cases that should be 'realistic'.
     cases = [
-        pytest.param(with_clay_and_grass.layers_dict, id="With grass and clay layer"),
-        pytest.param(
-            with_acceptance_criteria.layers_dict, id="With acceptance criteria"
-        ),
+        pytest.param(with_clay_and_grass.layers_dict, id="With grass and clay"),
+        pytest.param(with_acc_crit.layers_dict, id="With acc crit"),
```
Comment (Carsopre) [inline tests/acceptance_scenarios/layers_cases.py:51]: I do not see the benefit of renaming `acceptance criteria` to `acc crit` please revert.

File: tests/acceptance_scenarios/layers_cases.py [hunk 2]
```diff
@@ -34,7 +34,7 @@ class LayersCases(CasesProtocol):
             ],
         ),
     )
-    with_acceptance_criteria = LayersTestCase(
+    with_acc_crit = LayersTestCase(
```
Comment (Carsopre) [inline tests/acceptance_scenarios/layers_cases.py:37]: why this renaming? please revert.

File: tests/acceptance_scenarios/koswat_scenario_test_cases.py [hunk 3]
```diff
@@ -35,8 +35,8 @@ class ScenarioCases(CasesProtocol):
 
     cases = [
         pytest.param(default, id="Def Scenario"),
-        pytest.param(scenario_2, id="Scenario 2"),
-        pytest.param(scenario_3, id="Scenario 3"),
+        pytest.param(scenario_2, id="Scen 2"),
+        pytest.param(scenario_3, id="Scen 3"),
```
Comment (Carsopre) [inline tests/acceptance_scenarios/koswat_scenario_test_cases.py:39]: Why this renaming? please revert

**Review by Carsopre — APPROVED** [review]
Correct unnecessary renaming of tests (and therefore result files / dirs). After a green test bench it can be merged back to `master`.

File: tests/acceptance_scenarios/koswat_scenario_test_cases.py (hunk 3 above)
Comment (ArdtK) [reply tests/acceptance_scenarios/koswat_scenario_test_cases.py:39]: Some issue

File: tests/acceptance_scenarios/layers_cases.py (hunk 2 above)
Comment (ArdtK) [reply tests/acceptance_scenarios/layers_cases.py:37]: Same issue

File: tests/acceptance_scenarios/layers_cases.py (hunk 1 above)
Comment (Carsopre) [reply tests/acceptance_scenarios/layers_cases.py:51]: but we are going to end up having the same problem, modifying this "just here and there" will not solve the problem, if this is a recurring problem then we need to rethink how files are being generated (perhaps hashing the name). Given the current situation perhaps it's a good idea to address this problem in a separate issue.

Comment (ArdtK) [reply tests/acceptance_scenarios/layers_cases.py:51]: Reverted and created an issue https://github.com/Deltares/Koswat/issues/237

**Pull request merged**